src/_pytest/config/__init__.py

```diff
@@ -609,9 +615,6 @@ def _loadconftestmodules(
         if directory in self._dirpath2confmods:
             return
 
-        # XXX these days we may rather want to use config.rootpath
```

Comment:
This comment is outdated, we always configure `confcutdir` nowadays:

https://github.com/pytest-dev/pytest/blob/40011b838b67691c76302f3cd8d18862f37b9370/src/_pytest/config/__init__.py#L1334-L1339